package.json

```diff
@@ -15,12 +15,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "lite-server": "^2.4.0",
-    "truffle-hdwallet-provider-privkey": "^0.3.0"
+    "lite-server": "^2.4.0"
   },
   "devDependencies": {
-    "truffle-hdwallet-provider": "0.0.5",
-    "openzeppelin-solidity": "^1.12.0"
+    "@truffle/hdwallet-provider": "^1.1.1",
+    "openzeppelin-solidity": "^1.12.0",
+    "truffle": "4.1.9"
```

Comment:
The secret sauce was here, adding `truffle` to the dependency list (and using it to compile/migrate via `npx truffle migrate ...`.

Locked it at version `4.1.9` because that is a version that uses solidity version 0.4.24, which is what the Kudos contract uses.